script/setup/install-cni

```diff
@@ -21,7 +21,19 @@
 #
 set -eu -o pipefail
 
-CNI_COMMIT=${1:-$(grep containernetworking/plugins "$GOPATH"/src/github.com/containerd/containerd/go.mod | awk '{print $2}')}
+GOMODEnv=`go env |grep 'GOMOD=' | awk -F '=' '{print $2}' | sed 's/\"//g'`
```

Comment:
`source <(go env)` would make GOMOD and the rest of the go environment available directly. Is there a reason to not do this in this script, or at a higher level in the chain of calling scripts?

Reply:
`GOMOD` is a dynamic ENV, using `source <(go env)` adds complexity to reading `GOMOD` correctly.